crates/turborepo-lib/src/commands/login.rs

```diff
@@ -48,11 +41,11 @@ pub async fn sso_login(base: &mut CommandBase, sso_team: &str) -> Result<()> {
     let spinner = start_spinner("Waiting for your authorization...");
     direct_user_to_url(login_url.as_str());
 
-    let verification_token = Arc::new(OnceCell::new());
-    run_sso_one_shot_server(DEFAULT_PORT, verification_token.clone()).await?;
+    let token_cell = Arc::new(OnceCell::new());
```

Comment:
I know this is out of the scope of this PR, but this should really be a [`oneshot` channel](https://docs.rs/tokio/latest/tokio/sync/oneshot/index.html)

Reply:
Yeah, this is "future-diff-reduction" cleanup tagging along. I'll get that into the part where I collapse the two functions. 👍